addons/mail/static/src/messaging/component/chat_window_manager/chat_window_manager_tests.js

```diff
@@ -1014,6 +1017,72 @@ QUnit.test('open 3 different chat windows: not enough screen width', async funct
     );
 });
 
+QUnit.test('chat window: switch on TAB', async function (assert) {
+    assert.expect(7);
+
+    Object.assign(this.data.initMessaging, {
+        channel_slots: {
+            channel_channel: [{
+                channel_type: "channel",
+                id: 1,
+                name: "channel1",
+            }, {
+                channel_type: "channel",
+                id: 2,
+                name: "channel2",
+            }],
+        },
+    });
+    await this.start();
+    await afterNextRender(
+        () => this.env.entities.Thread.find(thread => thread.id === 1).open()
+    );
+
+    assert.containsOnce(document.body, '.o_ChatWindow', 'Only one chatWindow must be opened');
```

Comment:
- Double quote for assertion comments (reason: these strings are displayed on QUnit UI)
- Also could replace "one" by "1" (ease readability of assertion comment when it fails)